esmvaltool/cmorizers/data/formatters/datasets/esacci_lst.py

```diff
@@ -1,160 +1,138 @@
-
-            # Fix variable metadata
-            utils.fix_var_metadata(this_years_cubes, cmor_info)
-
-            # Fix global metadata
-            utils.set_global_atts(this_years_cubes, glob_attrs)
-
-            utils.save_variable(
-                this_years_cubes,
-                var_name,
-                out_dir,
-                glob_attrs,
-                unlimited_dimensions=['time'],
-            )
-
-
-def load_cubes(in_dir, file_day, file_night, year, month, variable):
-    """Variable description.
-
-    variable = land surface temperature
-    platform = AQUA not used for now
-               but in place for future expansion to all ESC CCI LST platforms
-    """
-    logger.info('Loading %s/%s%s%s*.nc', in_dir, file_day, year, month)
-    day_cube = iris.load_cube(
-        '%s/%s%s%02d*.nc' % (in_dir, file_day, year, month), variable)
-    logger.info('Loading %s/%s%s%s*.nc', in_dir, file_night, year, month)
-    night_cube = iris.load_cube(
-        '%s/%s%s%02d*.nc' % (in_dir, file_night, year, month), variable)
-
-    return day_cube, night_cube
-
-
-def make_monthly_average(day_cube, night_cube, year, month):
-    """Make the average LST form the day time and night time files."""
-    day_cube.attributes.clear()
-    night_cube.attributes.clear()
-
-    co_time = night_cube.coord('time')
-    co_time.points = co_time.points + 100.0
-    # maybe the arbitrary difference should go on day cubes to
-    # take the timestamp to 12Z?
-    # not really an issue when using monthly files
-
-    result = iris.cube.CubeList([day_cube, night_cube]).concatenate_cube()
-
-    # This corrects the lonitude coord name issue
-    # This should be fixed in the next version of the CCI data
-    logger.info("Longitude coordinate correction being applied")
-    result.coords()[2].var_name = 'longitude'
-    result.coords()[2].standard_name = 'longitude'
-    result.coords()[2].long_name = 'longitude'
-
-    monthly_cube = result.collapsed('time', iris.analysis.MEAN)
-
-    # fix time coordinate bounds
-    monthly_co_time = monthly_cube.coord('time')
-
-    time_point = (datetime.datetime(year, month, 1, 0, 0) -
-                  datetime.datetime(1981, 1, 1, 0, 0, 0)).total_seconds()
-    monthly_co_time.points = time_point
-
-    num_days = monthrange(year, month)[1]
-    monthly_co_time.bounds = [
-        time_point, time_point + ((num_days - 1) * 24 * 3600)
-    ]
-    # should this be num_days or num_days-1 ### question for Valeriu or Axel
-    # or 23:59:59 ???
-
-    monthly_cube.attributes = {
-        'information': 'Mean of Day and Night Aqua MODIS monthly LST'
-    }
-
-    return monthly_cube
+                try:
+                    cubes = load_cubes(in_dir,
+                                       vals['file'],
+                                       year,
+                                       month,
+                                       vals['raw'],
+                                       )
+                except:
```

Comment:
blank excepts are bad, do you know what sort of exception the iris cube loader may throw at you? It's not clear from the info printed below